deb/common/control

```diff
@@ -27,7 +27,7 @@ Vcs-Git: git://github.com/docker/docker.git
 
 Package: docker-ce
 Architecture: linux-any
-Depends: containerd.io (>= 1.2.2-3),
+Depends: containerd.io (>= 1.3.0),
```

Comment:
Need to check if these expect the `-x` suffix once we have packages for containerd v1.3.x available

Reply:
Discussing in our maintainers meeting; and should probably be fine without